doc/user_manual/code_interfaces/abce.tex

```diff
@@ -0,0 +1,150 @@
+    \item \textbf{demand\_data.csv}: normalized peak demand levels per simulated year (used to scale the \texttt{peak\_demand} parameter)
+    \item \textbf{unit\_specs.yml}: construction and operations cost and parameter data for all possible unit types in the model
+    \item \textbf{inputs/ts\_data/}:
+    \begin{itemize}
+      \item \textbf{timeseries\_<quantity>\_hourly.csv}: hourly timeseries data for each of the following quantities in the system:
+      \begin{itemize}
+        \item \textbf{load}: normalized to \texttt{peak\_demand}
+        \item \textbf{wind} and \textbf{solar}: wind and solar availability, normalized to the start-of-year installed capacity of each technology, respectively
+        \item \textbf{reg}, \textbf{spin}, and \textbf{nspin}: ancillary service procurement requirements, in absolute terms (not scaled)
+      \end{itemize}
+    \end{itemize}
+  \end{itemize}
+\end{itemize}
+
+
+
+The inputs in the \xmlNode{Files} section are shown below:
+
+\begin{lstlisting}[style=XML]
+    <Files>
+        <Input name="settings.yml" type="">settings.yml</Input>
+        <Input name="demand_data_file" type="" subDirectory="inputs">demand_data.csv</Input>
+        <Input name="agent_specifications_file" type="" subDirectory="inputs">single_agent_testing.yml</Input>
+        <Input name="unit_specs_data_file" type="" subDirectory="inputs">unit_specs.yml</Input>
+        <Input name="C2N_project_definitions.yml" type="" subDirectory="inputs">C2N_project_definitions.yml</Input>
+        <Input name="timeseries_nspin_hourly.csv" type="" subDirectory="inputs/ts_data">timeseries_nspin_hourly.csv</Input>
+        <Input name="timeseries_spin_hourly.csv" type="" subDirectory="inputs/ts_data">timeseries_spin_hourly.csv</Input>
+        <Input name="timeseries_reg_hourly.csv" type="" subDirectory="inputs/ts_data">timeseries_reg_hourly.csv</Input>
+        <Input name="timeseries_load_hourly.csv" type="" subDirectory="inputs/ts_data">timeseries_load_hourly.csv</Input>
+        <Input name="timeseries_wind_hourly.csv" type="" subDirectory="inputs/ts_data">timeseries_wind_hourly.csv</Input>
+        <Input name="timeseries_pv_hourly.csv" type="" subDirectory="inputs/ts_data">timeseries_pv_hourly.csv</Input>
+    </Files>
+\end{lstlisting}
+
+\subsubsection{Models}
+
+The \xmlNode{Code} model can be used with
+\xmlAttr{subType="Abce"} to run the ABCE Code Interface. The \xmlNode{executable} needs to be ABCE's executable \textbf{run.py}.
+
+\begin{lstlisting}[style=XML]
+  <Models>
+    <Code name="abce" subType="Abce">
+      <executable>abce/run.py</executable>
+      <clargs arg="python" type="prepend" />
+      <clargs arg="--settings_file" extension=".yml" type="input" delimiter="=" />
+      <clargs arg="--inputs_path=inputs --verbosity=3" type="text" />
+    </Code>
+  </Models>
+\end{lstlisting}
+
+\subsubsection{Output Files Conversion}
+
+The code interface reads in the \texttt{outputs\ABCE_run\outputs.xlsx}. 
```

Comment:
`\texttt{outputs\ABCE_run\outputs.xlsx}` should probably use `\verb` instead of `\texttt` so TeX does not think that `\ABCE` is a TeX command.

Reply:
Thanks!